recipes/vtk/all/conanfile.py

```diff
@@ -0,0 +1,139 @@
+from conans import ConanFile, CMake, tools
+import os
+import json
+
+
+class VtkConan(ConanFile):
+    name = "vtk"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://vtk.org/"
+    description = "VTK is an open-source software system for" \
+        "image processing, 3D graphics, volume rendering and visualization."
+    topics = ("conan", "vtk", "geometry", "algorithms")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False],
+               "build_testing": ["ON", "OFF", "WANT"],
+               "vtk_smp_implementation": ["Sequential", "OpenMP", "TBB", "STDThread"],
+               "vtk_wrap_python": [True, False],
+               "vtk_use_cuda": [True, False],
+               "vtk_use_mpi": [True, False],
+               "vtk_group_enable_qt": ["YES", "WANT", "DONT_WANT", "NO", "DEFAULT"],
```

Comment:
What is the difference between yes/want and no/dont_want? and default should be the default option?

Reply:
Yeah, I am not too clear about that as well. This just mirror the vtk cmake options. I think it has to do with priority level.

Reply:
Maybe just True/False keep it simple? Use something like https://github.com/conan-io/conan-center-index/blob/41d4fbb8c218f9d938ccb8b9d888bff52002e62e/recipes/wolfssl/all/conanfile.py#L99
to pass them if it's required